x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('AgentActivityFlyout', () => {
     ).toContain('Started on Sep 15, 2022 10:00 AM. Learn more'.replace(/\s/g, ''));
 
     act(() => {
-      fireEvent.click(result.getByText('Abort upgrade'));
+      fireEvent.click(result.getByText('Cancel'));
```

Comment:
renamed `Abort upgrade` to `Cancel` as suggested [here](https://github.com/elastic/kibana/issues/146018#issuecomment-1370025115)